hles/extraction/src/test/scala/org/broadinstitute/monster/dap/HLESExtractionPipelineBuilderIntegrationSpec.scala

```diff
@@ -69,4 +68,28 @@ class HLESExtractionPipelineBuilderIntegrationSpec extends PipelineBuilderSpec[A
   it should "successfully download records from RedCap" in {
     readMsgs(hlesOutputDir, "records/*.json") shouldNot be(empty)
   }
+
+  it should "contain records that all have a record, field_name, and value key" in {
+    readMsgs(hlesOutputDir, "records/*.json").foreach(msg => {
+      val record = msg.obj.toString()
+      record.contains("record") shouldBe true
+      record.contains("field_name") shouldBe true
+      record.contains("value") shouldBe true
+    })
+  }
+
+  it should "contain records with expected valid fields" in {
+    val allRecords = (readMsgs(hlesOutputDir, "records/*.json")).toString()
+    print(allRecords)
+    allRecords.contains("record") shouldBe true
```

Comment:
Same comment about not deserializing to string and instead comparing to a comparision upack `Obj`